lib/Exception/ValidationException.php

```diff
@@ -21,29 +21,37 @@
  *
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
  */
 
-namespace OCA\mail\tests\Unit\Service\Provisioning;
+namespace OCA\Mail\Exception;
+
+use Exception;
+use OCP\AppFramework\Http;
 
-use OCA\Mail\Service\Provisioning\Config;
+class ValidationException extends Exception {
+	/** @var array */
```

Comment:
```suggestion
	/** @var string[] */
```

for better static analysis